main.js

```diff
@@ -1,6 +1,8 @@
 var myTableEl = document.getElementsByName("table");
 var myHeaderEl = document.querySelector("span");
+var myForm = document.querySelector("form");

```

Comment:
As a general recommendation, I'd avoid using `my` in front of identifiers. While tutorials may use them frequently, they're much less useful in larger projects, and they usually get in the way of better names. For example, this `<form>` is used to create new grades, so something like `createGradeFormElement` would be more appropriate than `myForm`.